src/comp/comp.py

```diff
@@ -25,12 +25,18 @@ def __repr__(self):
 # whose name starts with 'D':
```

Comment:
The code looks good Michael 👍 , try and refactor these using the `comprehension` syntax:
`[ expression for item in list if conditional ]`
more info [here](https://www.pythonforbeginners.com/basics/list-comprehensions-in-python)